sdks/python/apache_beam/runners/portability/fn_api_runner/worker_handlers.py

```diff
@@ -737,11 +738,15 @@ def __init__(self,
 
   def host_from_worker(self):
     # type: () -> str
-    if sys.platform == "darwin":
+    if sys.platform == 'darwin':
       # See https://docs.docker.com/docker-for-mac/networking/
       return 'host.docker.internal'
-    else:
-      return super(DockerSdkWorkerHandler, self).host_from_worker()
+    if sys.platform != 'win32' and is_in_notebook():
```

Comment:
Why the special case for windows?

Reply:
The purpose is to only deal with linux-like system, so I could change this to sys.platform == 'linux' if we want to further limit the platform that need this change.

Reply:
Yeah I think we should check for linux if that's the goal, rather than relying on the fact that it's probably linux if it's not windows or mac.

Reply:
Changed to check for 'linux' platform.